lib/abide_dev_utils/cli/xccdf.rb

```diff
@@ -43,10 +43,11 @@ def execute(xccdf_file)
         end
         @data[:console] = true if @data[:dir].nil?
         @data[:type] = 'cis' if @data[:type].nil?
+        @data[:parent_key_prefix] = @data[:prefix]
```

Comment:
Why not just keep the assignment to `@data[:parent_key_prefix]` in the first place?

Reply:
Yeah, I guess that was kinda silly.